src/main/java/jenkins/plugins/clangscanbuild/actions/ClangScanBuildAction.java

```diff
@@ -1,23 +1,21 @@
 package jenkins.plugins.clangscanbuild.actions;
 
-import hudson.FilePath;
-import hudson.model.Action;
-import hudson.model.ModelObject;
-import hudson.model.AbstractBuild;
-import hudson.util.FormValidation;
-
 import java.io.IOException;
+import java.util.logging.Logger;
 import java.util.regex.Pattern;
 
-import javax.servlet.ServletException;
+import org.kohsuke.stapler.StaplerProxy;
```

Comment:
@escoem No? I thought you had adopted this plugin. In this case, it is interesting to mention the maintainer in the PR.